lte/gateway/c/core/oai/test/mme_app_task/CMakeLists.txt

```diff
@@ -48,12 +48,17 @@ set(NAS_APN_AMBR
 
 set(MME_APP_TEST_SRC
```

Comment:
My understanding is that adding the tests here is sufficient to executed the extracted tests via `mme_app_test.cpp`. Other tests are configured as single runable test targets. -> Are currently tests in CI executed double via make? By `mme_app_test.cpp` and as a single target?